### PR TITLE
21 esp32 http support

### LINKED ISSUE #21: ESP32 http support

### PR DESCRIPTION
**Pull request by Bissas**
# New features
- HTTP support for ESP32 

# Fixes 
- Updated url paths 
- Updated arduino-lint CI

### CODE REVIEW

File: src/handler/deploii_handler.h [hunk 1]
```diff
@@ -13,7 +16,11 @@
 
 #define Deploii_HOST "deploii.no"
 #define Deploii_PORT 443
-#define Deploii_WS_URL "/ws"
+#define Deploii_WS_URL "/mcu/ws"
+#define Deploii_HTTP_URL "/mcu/data"
```
Comment (sebwk01) [inline src/handler/deploii_handler.h]: ikke mcu/http?

File: src/handler/deploii_handler.h [hunk 2]
```diff
@@ -13,7 +16,11 @@
...
+
+#define Deploii_WIFI_RECONNECT_TIME 1000
```
Comment (sebwk01) [inline src/handler/deploii_handler.h]: DEPLOII eller Deploii?

File: src/handler/deploii_handler.h (hunk 1 above)
Comment (Bissas) [reply src/handler/deploii_handler.h]: Needs to correspond to the back end as well. Will probably be changed in a later PR.

**Review by erlestor — APPROVED** [review]
LGTM!

**Pull request merged**